fixes #10 - new Blob() - Illegal constructor

pheinicke opened the pull request:
- use BlobBuilder if creating blob via constructor fails (as it happens on Android 4.3)

markmarijnissen commented:
Sadly, you've changed too much for me to merge. 

I've started merging pull requests to maintain this project (at the bare minimum). Would you be willing to make a new pull request which only fixes #10?

pheinicke commented:
Well, it's been quite some time now - not sure how this got to be a PR... It sure was not intended to be one. Since you did not have time back then, I just forked the project and adjusted it to my own needs.

I've extracted the fix for #10 into a separate PR...
